src/Domain/JapaneseParser.php

```diff
@@ -97,4 +97,29 @@ public function getParsedTokens(string $text, Language $lang) {
         return $tokens;
     }
 
+    public function getReading(string $text, Language $lang) {
```

Comment:
Most of this method deals with opening the file and writing the content.  The method `getParsedTokens()` already has a lot of that, so this could probably be refactored to extract a common method that both of them use, like:

```
private function getMecabResult(string $text, string $args): string {
  // create and open a temp file
  // write result
  // read and return full file
}

Then getParsedTokens and getRomanization would use that method.